x/did/tests/create_diddoc_test.go

```diff
@@ -181,6 +181,8 @@ var _ = Describe("Create DID tests", func() {
 					Id:              did + "#service-1",
 					Type:            "type-1",
 					ServiceEndpoint: []string{"endpoint-1"},
+					Accept:          []string{"accept-1"},
+					RoutingKeys:     []string{"did:canow:testnet:HPXoCUSjrSvWC54SLWQjsm"},
```

Comment:
`RoutingKeys` should contain DID Fragment URL instead of just DID

Reply:
Done! ✅